.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Keep Gradle wrapper distribution as is
+/gradlew                            binary
+/gradlew.bat                        binary
```

Comment:
GitHub ignores this and still shows the diff. Could be confusing when checking things locally.

Reply:
Ok. I want to prevent Git from making text conversations (i.e. changing the line terminator). Would you like it if I replace `binary` with `-text`? Or do you have another suggestion?